main.js

```diff
@@ -0,0 +1,30 @@
+$('.submit-post').on('click', function () {
+  var $input = $('#input-post').val();
+  var $name = $('#input-name').val();
+
+  $('.posts').append('<div> <button type="button" class="btn btn-link comment">comment</button> ' + $input + ' -Posted by: ' + $name + '<button type="button" class="btn btn-link remove">x</button> <div class="comment-section" style="display: none"> <div class="comments"> </div> <div ="comment-input-area"> <input type="text"class="form-control input-area input-comment" placeholder="Your comment"/> <input type="text"class="form-control input-area input-name2" placeholder="Your name"/> <button type="button" class="btn btn-primary submit-comment"> Submit Comment </button> </div>  </div> <hr /> </div> ');
+  bindRemove();
+});
+
+$('.posts').on('click', '.comment', function () {
+  var $commentSection =  $(this).closest('div').find('.comment-section');
+  if ($commentSection.css('display') ==='none') {
+    $commentSection.show();
+  }else {
+    $commentSection.hide();
+  }
+});
+
+$('.posts').on('click', '.submit-comment', function () {
+  var $commentInput = $(this).closest('.comment-section').find('.input-comment').val();
```

Comment:
The chaining of all these jQuery methods is fine, but I'd want you to consider other strategies as well because - what if the DOM changes? This code would no longer work.